src/index.ts

```diff
@@ -0,0 +1,4 @@
+export type { VueDriftWidgetConfig, DriftWidget } from './types'
+export { createDrift } from './install'
```

Comment:
maybe this should be default to align with other plugins?

Reply:
you mean with vue2 plugins? I don't see this being used in vue3 plugins tho?

Reply:
https://v3.vuejs.org/guide/plugins.html#writing-a-plugin done in the example.. that's why... also true for apex charts etc.

Reply:
changed in f55c995